x-pack/test/functional/services/ml/data_visualizer_table.ts

```diff
@@ -565,6 +565,18 @@ export function MachineLearningDataVisualizerTableProvider(
       }
     }
 
+    public async assertLensActionShowChart(fieldType: string, fieldName: string) {
+      await retry.tryForTime(10 * 1000, async () => {
+        await testSubjects.clickWhenNotDisabled(
+          this.rowSelector(fieldName, 'dataVisualizerActionViewInLensButton')
+        );
+        await testSubjects.existOrFail('lnsVisualizationContainer', {
+          timeout: 1000,
```

Comment:
When running the tests with `TEST_THROTTLE_NETWORK=1` to simulate a slower network environment, this failed for me as the Lens page took longer to load. I'd suggest to make this a 15 seconds /  `15 * 1000` timeout and increase the retry timeout to 30 seconds.